### DISCUSSION TITLE
Github Messes Up Valid Links

### DISCUSSION THREAD

**Discussion by machineghost** [community/community#20240] [Other Feature Feedback, Questions, & Ideas] [answered]
I recently tried to link the following Giithub URL in an issue: `https://github.com/keystonejs/keystone/discussions/` + `7696`.

Github tried to convert, incorrectly, into a link to a pull request ... and even removed a digit to do so (it linked to #769 in that repository).

Obviously, if I link to a discussions page on GitHub, it should *remain a link to the discussions page* ... not become a link to a PR of a different number.

**Comment by edhends** [comment, marked as answer]
Hi @machineghost you can report bug here! https://support.github.com/contact/bug-report
